libraries/AP_Follow/AP_Follow.cpp

```diff
@@ -340,19 +351,51 @@ bool AP_Follow::handle_global_position_int_message(const mavlink_message_t &msg)
         if ((packet.lat == 0 && packet.lon == 0)) {
             return false;
         }
+        // ignore message if we already received a more recent one
+        if(_target_location_last_time_boot_ms != 0 && packet.time_boot_ms <= _target_location_last_time_boot_ms ) {
```

Comment:
What happens if the follow target vehicle is rebooted? Its boot time will go backwards so it will be ignored. Maybe if the time is more than 5 or 10 seconds in the past it would reset the boot time.